core/domain/exp_domain_test.py

```diff
@@ -2103,7 +2110,7 @@ class SchemaMigrationUnitTests(test_utils.GenericTestBase):
       fallbacks: []
       hints: []
       id: TextInput
-      solution: {}
+      solution: null
```

Comment:
Hm, but quite a lot of backend tests were failing on Travis at the time I did this review. Did they not fail on your computer? If so, that's something you might want to investigate...

Btw, I also noticed that the current PR has merge conflicts.

EDIT: Also, I'm slightly confused. The current comment seems to be appearing on a schema that's not the v13 version. I'm not sure what happened there; maybe this is some GitHub auto-magic :)